.github/workflows/test.yml

```diff
@@ -164,7 +134,7 @@ jobs:
   # When adding new jobs, please add them to `needs` below
   check-tests:
     name: "check tests"
-    needs: [ci, std, integration, doc, website, scala-cli-template]
+    needs: [test, mima, doc, website, scala-cli-template]
```

Comment:
Splitting out `mima` from CI is a good idea, but do not combine `ci`, `std`, and `integration`, they are split on purpose because then they can run in parallel on separate runners and CI is a lot faster.

Reply:
I see, I think integration can keep, but `std` seem only exist one test， so I prefer moving it to integration.